crawler-cli/src/test/java/org/gbif/crawler/pipelines/dwca/DwcaToAvroCallBackTest.java

```diff
@@ -0,0 +1,74 @@
+package org.gbif.crawler.pipelines.dwca;
+
+import org.gbif.api.model.crawler.DwcaValidationReport;
+import org.gbif.api.vocabulary.DatasetType;
+import org.gbif.common.messaging.api.messages.DwcaValidationFinishedMessage;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.util.UUID;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileUtil;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.MiniDFSCluster;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Test dwca-to-avro commands message handling command on hdfs
+ */
+public class DwcaToAvroCallBackTest {
+
+
+  private final static String DATASET_UUID_POS = "9bed66b3-4caa-42bb-9c93-71d7ba109dad";
```

Comment:
static final instead of final static, the for the next lines

Reply:
done